web_poet/page_inputs/stats.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, Dict, Protocol
+
+
+class StatCollectorProtocol(Protocol):
```

Comment:
It seems it'd be easier to have an abc for stats collector, not a protocol. 
Also, it seems the protocol is currently unused?